.bazelci/presubmit.yml

```diff
@@ -31,8 +31,8 @@ tasks:
   integration_tests:
     name: "Integration Tests"
     platform: ${{ integration_platform }}
-    test_flags:
-      - "--config=rbe"
+#    test_flags:
```

Comment:
Did you mean to leave this one commented out still?

Reply:
Fixed.